docker/scripts/dev_start.sh

```diff
@@ -278,10 +276,6 @@ function setup_devices_and_mount_local_volumes() {
     setup_device
 
     local volumes="-v $APOLLO_ROOT_DIR:/apollo"
-    local teleop="${APOLLO_ROOT_DIR}/../apollo-teleop"
```

Comment:
No longer mount local `teleop`  repo for the OSS version.

Reply:
why skip the teleop mounting step?

Reply:
> why skip the teleop mounting step?

It was mounted in our internal `dev_start.sh`, and teleop was not public visible.
Maybe did the mount here target teleop guys ?

Reply:
> > why skip the teleop mounting step?
> 
> It was mounted in our internal `dev_start.sh`, and teleop was not public visible.
> Maybe did the mount here target teleop guys ?

yes, to make things easier for Udelv folks.

Reply:
OK. Restored. Sorry for my unthoughtfulness.